practices/code-review.md

```diff
@@ -0,0 +1,34 @@
+# Code Reviews
+
+Code Review, or Peer Code Review, is the act of consciously and systematically checking the code or your colleagues for mistakes and errors. It has been repeatedly shown to accelerate and streamline the process of software development.
```

Comment:
"consciously and" < redundant. You can't unconsciously check anything.
"the code or your colleagues"?